packages/types/src/fulu/sszTypes.ts

```diff
@@ -0,0 +1,258 @@
+  {
+    blockRoot: Root,
+    index: ColumnIndex,
+  },
+  {typeName: "DataColumnIdentifier", jsonCase: "eth2"}
+);
+
+export const DataColumnSidecarsByRootRequest = new ListCompositeType(
+  DataColumnIdentifier,
+  MAX_REQUEST_DATA_COLUMN_SIDECARS
+);
+
+export const DataColumnSidecarsByRangeRequest = new ContainerType(
+  {
+    startSlot: Slot,
+    count: UintNum64,
+    columns: new ListBasicType(ColumnIndex, NUMBER_OF_COLUMNS),
+  },
+  {typeName: "DataColumnSidecarsByRangeRequest", jsonCase: "eth2"}
+);
+
+export const ExecutionPayload = new ContainerType(
+  {
+    ...electraSsz.ExecutionPayload.fields,
+  },
+  {typeName: "ExecutionPayload", jsonCase: "eth2"}
+);
+
+export const ExecutionPayloadHeader = new ContainerType(
+  {
+    ...electraSsz.ExecutionPayloadHeader.fields,
+  },
+  {typeName: "ExecutionPayloadHeader", jsonCase: "eth2"}
+);
+
+export const BeaconBlockBody = new ContainerType(
+  {
+    ...electraSsz.BeaconBlockBody.fields,
+  },
+  {typeName: "BeaconBlockBody", jsonCase: "eth2", cachePermanentRootStruct: true}
+);
+
+export const BeaconBlock = new ContainerType(
+  {
+    ...electraSsz.BeaconBlock.fields,
+  },
+  {typeName: "BeaconBlock", jsonCase: "eth2", cachePermanentRootStruct: true}
+);
+
+export const SignedBeaconBlock = new ContainerType(
+  {
+    message: BeaconBlock,
+    signature: BLSSignature,
+  },
+  {typeName: "SignedBeaconBlock", jsonCase: "eth2"}
+);
+
+export const BlobSidecar = new ContainerType(
+  {
+    ...denebSsz.BlobSidecar.fields,
+  },
+  {typeName: "BlobSidecar", jsonCase: "eth2"}
+);
+
+export const BlobsBundle = new ContainerType(
+  {
+    commitments: denebSsz.BlobKzgCommitments,
+    proofs: KZGProofs,
+    blobs: denebSsz.Blobs,
+  },
+  {typeName: "BlobsBundle", jsonCase: "eth2"}
+);
+
+export const BlindedBeaconBlockBody = new ContainerType(
+  {
+    ...electraSsz.BlindedBeaconBlockBody.fields,
+  },
+  {typeName: "BlindedBeaconBlockBody", jsonCase: "eth2", cachePermanentRootStruct: true}
+);
+
+export const BlindedBeaconBlock = new ContainerType(
+  {
+    ...electraSsz.BlindedBeaconBlock.fields,
+  },
+  {typeName: "BlindedBeaconBlock", jsonCase: "eth2", cachePermanentRootStruct: true}
+);
+
+export const SignedBlindedBeaconBlock = new ContainerType(
+  {
+    message: BlindedBeaconBlock,
+    signature: BLSSignature,
+  },
+  {typeName: "SignedBlindedBeaconBlock", jsonCase: "eth2"}
+);
+
+export const BuilderBid = new ContainerType(
+  {
+    ...electraSsz.BuilderBid.fields,
+  },
+  {typeName: "BuilderBid", jsonCase: "eth2"}
+);
+
+export const SignedBuilderBid = new ContainerType(
+  {
+    message: BuilderBid,
+    signature: BLSSignature,
+  },
+  {typeName: "SignedBuilderBid", jsonCase: "eth2"}
+);
+
+export const ExecutionPayloadAndBlobsBundle = new ContainerType(
+  {
+    executionPayload: ExecutionPayload,
+    blobsBundle: BlobsBundle,
+  },
+  {typeName: "ExecutionPayloadAndBlobsBundle", jsonCase: "eth2"}
+);
+
+export const BeaconState = new ContainerType(
```

Comment:
do we actually need to add all of these even if unchanged?

eg. in Deneb we still reference `phase0.Attestation`

https://github.com/ChainSafe/lodestar/blob/30d888c61292cbd19f49f062e77c6e2b4f62dde1/packages/types/src/types.ts#L185

Reply:
yeah these should all be deleted, only unique types added here